README.md

```diff
@@ -36,6 +36,9 @@ Read [this blog post on functional ASP.NET Core](https://dusted.codes/functional
 
 If you'd like to learn more about the motivation of this project please read my [blog post on functional ASP.NET Core](https://dusted.codes/functional-aspnet-core) (some code samples in this blog post might be outdated today).
 
+> [!TIP]
+> Check the [release notes](./RELEASE_NOTES.md) document to know how the system is evolving, and to get more information about breaking changes.
+
```

Comment:
I've never seen `[!TIP]` syntax, but it renders nicely! It helps with this standing out, looks good!

Reply:
It was added by the end of last year, apparently. I discovered about it after reading this project from @mrtz-j: https://github.com/giraffe-fsharp/Giraffe.OpenApi.

Check this issue for more context on this feature discussion: https://github.com/orgs/community/discussions/16925.